js/helpers/commonmark.js

```diff
@@ -1,9 +1,9 @@
 import markdownit from 'markdown-it';
 import { options } from 'markdown-it/lib/presets/commonmark';
 
+options.linkify = true;
+const markdown = markdownit(options);
 
 export default function(text) {
```

Comment:
An alternative to the default() is https://www.npmjs.com/package/babel-plugin-add-module-exports but it will add some more code to the generated output.